tests/src/Nightwatch/Commands/paragraphs/add.js

```diff
@@ -28,7 +28,6 @@ exports.command = function add(fieldName, type, position) {
     .scrollIntoMiddleOfView(
       `//table[contains(@id, "${fieldNameId}-values")]/tbody/tr[${addButtonPosition}]//input`
     )
-    .pause(10000)
```

Comment:
Amazing performance improvement right here...

Reply:
`4x` improvement. I doubt we will ever beat that.